framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -230,25 +242,38 @@ public int hashCode() {
         }
     }
 
-    public List<Contract> getContracts(ExecutableElement element) {
+    /**
+     * Returns all the contracts on {@code methodElement}.
+     *
+     * @param methodElement the method or constructor whose contracts to retrieve
```

Comment:
Would `executableElement` be a better name? I find `methodElement` misleading for something that can be either a method or a constructor.

Reply:
This would be a reasonable change.  A search revealed that this is a widespread issue in the Checker Framework, not isolated to this class.